sdks/RepoToolset/tools/Build.proj

```diff
@@ -185,15 +176,6 @@
              BuildInParallel="true"
              Condition="'@(_SolutionBuildTargets)' != ''" />
 
-    <!--
-      Workaround for https://github.com/NuGet/Home/issues/6165.
```

Comment:
I presume that this workaround will still be needed if the toolset is going to (potentially) be consumed in a repo using an older version of the NuGet client.  Probably not an issue for RepoToolset repos but it may be for the arcade tools?

Reply:
Not sure what do you mean. This build project is only gonna be used in repos using RepoToolset. It implements the CI build script for these repos. Also I don't think we should support older NuGet clients in any of our repos. They should all be on the latest released build.

Reply:
The comment was meant merely to confirm my assertion that the problem has been fixed in newer versions of the NuGet client, but if someone attempted to use RepoToolset, or some other toolset with similar semantics, in a servicing branch, then the problem would still exist.  It was a general comment and not specifically applicable to this PR.  

Thanks for making the SDK change!

Reply:
Oh, I see. Yes, it would exists in older branches.